k8s/linux-amd64/Dockerfile

```diff
@@ -44,8 +44,13 @@ RUN curl --silent -L "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip"
 # Get gke-gcloud-auth-plugin (reqd for kubectl 1.26+)
 # See https://github.com/OctopusDeploy/Issues/issues/7621 for more info
 RUN curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && \
-    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
-    apt-get update && apt-get install -y google-cloud-sdk-gke-gcloud-auth-plugin
+    #echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \

```

Comment:
Can we remove the commented lines please :)

Reply:
Done